magicctapipe/io/tests/test_io_monly.py

```diff
@@ -17,456 +19,564 @@
 )
 
 
-def test_format_object():
-    """
-    Simple check on a string
-    """
-    str_a = "a{b[[xz,cde}"
-    str_b = format_object(str_a)
-    assert str_b == "a b  xzcde "
-
+class TestGeneral:
+    def test_format_object(self):
```

Comment:
the same function is tested in test_io.py, so I do not think you need to test it again here

Reply:
removed